### PR TITLE
Rename `rollup-template` to `sov-rollup-starter`

### PR DESCRIPTION
**Pull request by bkolad**
# Description
Renaming discussed with @cemozerr.

## Linked Issues
- Fixes # (issue, if applicable)
- Related to # (issue) 

## Testing
No changes.

## Docs
No changes.

### CODE REVIEW

File: sov-rollup-starter/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+publish.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (vlopes11) [inline sov-rollup-starter/Cargo.toml:12]: ni: consider taking this out

File: sov-rollup-starter/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+[dependencies]
+#stf dependencies
+sov-modules-api = { path = "../module-system/sov-modules-api", features = ["native"] }
```
Comment (vlopes11) [inline sov-rollup-starter/Cargo.toml:16]: question: shouldn't `native` be an optional dependency? In case the user wants his own primitives

**Review by vlopes11 — APPROVED** [review]
Looks good! A couple of optional questions/nits

File: sov-rollup-starter/Cargo.toml (hunk 2 above)
Comment (bkolad) [reply sov-rollup-starter/Cargo.toml:16]: This is intended `sov-rollup-starter` is a leave crate that runs the rollup so it always needs `native`

File: sov-rollup-starter/Cargo.toml (hunk 1 above)
Comment (bkolad) [reply sov-rollup-starter/Cargo.toml:12]: Good catch, I will in the next PR.

**Pull request merged**